### ISSUE TITLE
[UX] Move updates settings under the new consolidated "System updates" page

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#5084]
With #2714 we now have a unified page where people can check for and perform updates, under Configuration -> System -> System updates (`admin/config/system/updates`). Settings for these updates though are still located under Reports -> Updates -> Settings (`admin/reports/updates/settings`).

I think that no settings page should be under "Reports" (the updates settings page is the only one currently where this happens), so I propose that we move the updates settings page under Configuration -> System -> System updates -> Settings (`admin/config/system/updates/settings`) instead. In general, it makes sense to configure settings under the same place where the thing the settings apply to lives.

**Comment by klonos** [comment]
Hmm 🤔 ...apparently this would require the Installer module to be responsible for the updates settings page, whereas now the Update module is responsible for it (as well as the "Available updates" report).

**Comment by klonos** [comment]
...well not entirely true. The "Check for available updates" and "Update Notifications" sections under `admin/reports/updates/settings` hold settings relevant to the Update module, whereas the "Self-updates" section holds settings for the Installer module 🤔 🤔 🤔

**Comment by klonos** [comment]
PR up for review: https://github.com/backdrop/backdrop/pull/3622

The "Self-updates" fieldset from `admin/reports/updates/settings` is now moved under a new `admin/config/system/updates/settings` form:

![image](https://user-images.githubusercontent.com/2423362/118023114-d2860780-b365-11eb-9ee8-4b1ac1da0533.png)

Now these 2 forms save settings that belong to the respective module.

**Comment by jenlampton** [comment]
@klonos I don't think this is an improvement. As a user, I don't want to have my "Update settings" spread across two different forms. I don't care which module is responsible for which values, it's far easier to only need to find one obscure location to enter my settings.
